Index: src/com/admin/access/AdminAccess.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.admin.access;\r\n\r\npublic class AdminAccess {\r\n}\r\n
===================================================================
--- src/com/admin/access/AdminAccess.java	(revision 98143fadf10e03f9161f2237779e347a0b0b6668)
+++ src/com/admin/access/Films.java	(date 1586961541025)
@@ -1,4 +1,10 @@
 package com.admin.access;
 
-public class AdminAccess {
+import java.util.ArrayList;
+
+public abstract class Films {
+
+
+
+
 }
